lib/Config.ts

```diff
@@ -151,6 +167,13 @@ class Config {
     this.dbpath = this.getDefaultDbPath();
   }
 
+  private updateMacaroonPaths = (network: string) => {
+    this.network = network as Network;
+    this.lndbtc.macaroonpath = path.join(this.lndbtc.macaroonpath, '..', '..', this.network, 'admin.macaroon');
+    this.lndltc.macaroonpath = path.join(this.lndltc.macaroonpath, '..', '..',
+      this.network === Network.TestNet ? 'testnet4' : this.network, 'admin.macaroon');
```

Comment:
I would put `this.network === Network.TestNet ? 'testnet4' : this.network` into a method since it is used twice and pretty easy to get wrong.

Reply:
It sort of is in a method, `updateMacaroonsPath`. I can't use it in the constructor because it's the first time we're setting a value for `macaroonpath`, but if the logic is ever needed to be used again it can call this method again. Also I think longer term this may need to change for a more generic way to handle things - e.g. what if there's a `testnet5` one day or a separate currency supported by lnd one day has its own numbering system. I was thinking reading from the directory and using whichever directory starts with "testnet" maybe.

FYI, I think it's actually pretty annoying that the testnet folder is different for litecoin in lnd, especially given that within the code and config (namely the `--litecoin.testnet` option) it just uses "testnet" most places, so I'm also hoping that one day lnd just uses a plain "testnet" folder. Was thinking of opening an issue over there about this.

Long story short, I can definitely still put this in its own method if you'd like, but I feel it's OK for now as is.

Reply:
No it is fine as is then.

#554 should take care of the completely dynamic approach then.